queries.sql

```diff
@@ -9,3 +9,46 @@ SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
 
+update animals set species = name where name like '%mon%'; 
+update animals set species = 'pokemon' where species is null;
```

Comment:
- [ ] Please we should write the query to update the animal name in the `data.sql`.